.dockerignore

```diff
@@ -1,4 +1,8 @@
-*
-!requirements
-!requirements-python3
-!package.json
+__pycache__/
```

Comment:
Is this line supposed to match any directory with that name anywhere in the repo, or only the one at the root? All of the paths in this file match files in the root. A `**/` prefix is required to match files at arbitrary places in the tree.

Reply:
Thank you for spotting that! d0ea8036edd